app/javascript/packs/App.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import Body from './Body'
+import MyHeader from './MyHeader'
+
+class App extends React.Component {
+
+    constructor() {
+        super();
```

Comment:
You don't need a constructor here since it's empty and only sets the state. You can do this instead:
`state = {
    cuisines: []
}
`